modules/sku/src/ProductManager.php

```diff
@@ -564,7 +576,7 @@ public function synchronizeStockData(array $stock = [], $storeId = '') {
       'success' => TRUE,
     ];
 
-    return (new ResourceResponse($response));
+    return ($response);
```

Comment:
please change this back, it needs to be ResourceResponse because of caching

Reply:
I wondered about that. Here the `$response` is returned as an array into `\Drupal\acm_sku\Plugin\rest\resource\ProductSyncResource::post`where the response is formed and sent back.

There are two resources, yes? In `\Drupal\acm_sku\Plugin\rest\resource\ProductSyncResource::post` Should I use: (for non-cached I think)
1. `Drupal\rest\ModifiedResourceResponse;`
or (for cached, yes?)
2. `Drupal\rest\ResourceResponse;`

We should always use 2., right?